Add missing step in the MC template (#17)
giantswarm/gitops-template: docs/add_mc.md

    EOF
    ```

> 2. Create the MC kustomization file poiting the git repo and applied it:

Author: ```suggestion
2. Create the MC kustomization file pointing to the git repo and apply it:
```

> 2. Create the MC kustomization file pointing to the git repo and apply it:

Reviewer: ```suggestion
1. Create the MC kustomization file pointing to the git repo and apply it:
```

> 1. Create the MC kustomization file pointing to the git repo and apply it:

Reviewer 2: ```suggestion
1. Apply the cluster's Kustomization CR:

```sh
kubectl apply -f management-clusters/${MC_NAME}/${MC_NAME}.yaml
```

Reviewer 2: we do not need to create this Kustomization CR since it leaves in the repository now, we just need to apply it
